stylesheets/stylesheets/scrollbar.css

```diff
@@ -0,0 +1,30 @@
+html.dark {
+    --theme-scrollbar-thumb: #202225;
+    --theme-scrollbar-track: #2f3136;
+}
+
+html.light {
```

Comment:
Light theme doesn't actually have a class - Just remove the .light here and it looks great!

Reply:
Cheers, glad I could help. I was running some tests earlier and found that `light` on the html element occurs as a class. Although the css design would still work anyways without the light class label as in dark theme, the dark class overrides the element theme.![image](https://user-images.githubusercontent.com/63494857/132132346-04a897ad-cc4c-4025-95aa-f843bed95d4f.png)

Reply:
huuuuuuuh. This must be a byproduct of the extension I installed, didn't know it does this